app/src/main/java/com/goldenraven/padawanwallet/ui/wallet/SendScreen.kt

```diff
@@ -281,15 +280,55 @@ fun Dialog(
     }
 }
 
-private fun broadcastTransaction(recipientAddress: String, amount: ULong, feeRate: Float = 1F) {
+fun verifyInput(
+    address: String,
+    amount: String,
+    feeRate: String,
+    snackbarHostState: SnackbarHostState,
+    scope: CoroutineScope,
+): Boolean {
+    if (address == "") {
+        scope.launch {
+            snackbarHostState.showSnackbar(message = "Address is missing!", duration = SnackbarDuration.Short)
+        }
+        return false
+    }
+    if (amount == "") {
+        scope.launch {
+            snackbarHostState.showSnackbar(message = "Amount is missing!", duration = SnackbarDuration.Short)
+        }
+        return false
+    }
+    if (feeRate != "" && (feeRate.toInt() <= 1 || feeRate.toInt() > 200)) {
```

Comment:
This check should actually accept 1 (but nothing below).

`if (feeRate != "" && (feeRate.toInt() < 1 || feeRate.toInt() > 200))` would do it.

Reply:
Same comment as above for using `isBlank()`.